### PR TITLE
Azure Compute Fleet GA version 2024-11-01 Minor update to Documentation only.

### PR DESCRIPTION
**Pull request by rahuls-microsoft**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

Azure Compute Fleet GA version 2024-11-01

### CODE REVIEW

File: eng/Packages.Data.props
Comment (jsquire) [inline eng/Packages.Data.props:357]: This change already exists in main and should not appear in this PR.

**Review by jsquire — DISMISSED** [review]
@rahuls-microsoft :

Your account lacks the required public Azure GitHub organization required of an internal contributor.  Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.

- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

File: eng/Packages.Data.props
Comment (rahuls-microsoft) [reply eng/Packages.Data.props:357]: Ok, I was getting build errors. I will pull the changes from main.

**Comment by rahuls-microsoft** [comment]
> @rahuls-microsoft :
> 
> Your account lacks the required public Azure GitHub organization required of an internal contributor. Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
> 
> * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_
> 
> You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.
> 
> Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

Added myself to Codeowners.

File: eng/Packages.Data.props
Comment (rahuls-microsoft) [reply eng/Packages.Data.props:357]: I have merged from the source branch, so it should resolve as no change, right?

Comment (jsquire) [reply eng/Packages.Data.props:357]: Rebase on main, don't merge.   You want to put your changes on top of the latest.   There should be no files in this PR that you're not changing for your work.

**Comment by rahuls-microsoft** [comment]
> > @rahuls-microsoft :
> > Your account lacks the required public Azure GitHub organization required of an internal contributor. Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
> > 
> > * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> > * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_
> > 
> > You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.
> > Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.
> 
> Added myself to Codeowners. But gives error on adding it. How do add myself to CODEOWNERS - directly checkin to - https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS ?

https://github.com/Azure/azure-sdk-for-net/pull/46554 - Created a pull request to add myself to codeowners for Fleet.

File: eng/Packages.Data.props
Comment (rahuls-microsoft) [reply eng/Packages.Data.props]: rebased to https://github.com/Azure/azure-sdk-for-net.git

**Comment by rahuls-microsoft** [comment]
> @rahuls-microsoft :
> 
> Your account lacks the required public Azure GitHub organization required of an internal contributor. Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
> 
> * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_
> 
> You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.
> 
> Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

Added myself to - https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS

**Comment by rahuls-microsoft** [comment]
@jsquire - Updated the PR after rebase.
Please review.

**Comment by jsquire** [comment]
> @jsquire - Updated the PR after rebase. Please review.

Common repository files look good.   You'll need Arthur and team for the review of your library code.   Your group memberships are still incorrect, and this will remain blocked until we fix those.

File: .vscode/cspell.json
Comment (danieljurek) [inline .vscode/cspell.json:178]: Please insert this into the list in alphabetical order

Comment (rahuls-microsoft) [reply .vscode/cspell.json:178]: ok

**Comment by rahuls-microsoft** [comment]
> > @jsquire - Updated the PR after rebase. Please review.
> 
> Common repository files look good. You'll need Arthur and team for the review of your library code. Your group memberships are still incorrect, and this will remain blocked until we fix those.

I added myself in the main, should I add myself in this PR? Sorry not familiar with the process. Thanks for your help.

**Comment by rahuls-microsoft** [comment]
The PS scripts gives errors-
PS C:\src\Fleet-NetSDK\sdk\computefleet\Azure.ResourceManager.ComputeFleet> ..\..\..\eng\scripts\Export-API.ps1
At C:\src\Fleet-NetSDK\eng\scripts\Export-API.ps1:19 char:57
+ $diagnosticArguments = ($debugLogging -and $logsFolder) ? "/binarylog ...
+                                                         ~
Unexpected token '?' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParseException
    + FullyQualifiedErrorId : UnexpectedToken

PS C:\src\Fleet-NetSDK\sdk\computefleet\Azure.ResourceManager.ComputeFleet> ..\..\..\eng\scripts\CodeChecks.ps1 -ServiceDirectory computefleet
At C:\src\Fleet-NetSDK\eng\scripts\CodeChecks.ps1:97 char:65
+ ...     $diagnosticArguments = ($debugLogging -and $logsFolder) ? "/binar ...
+                                                                 ~
Unexpected token '?' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParseException
    + FullyQualifiedErrorId : UnexpectedToken
    
    Any help on this?

**Comment by rahuls-microsoft** [comment]
> > > @jsquire - Updated the PR after rebase. Please review.
> > 
> > 
> > Common repository files look good. You'll need Arthur and team for the review of your library code. Your group memberships are still incorrect, and this will remain blocked until we fix those.
> 
> I added myself in the main, should I add myself in this PR? Sorry not familiar with the process. Thanks for your help.

https://github.com/Azure/azure-sdk-for-net/pull/46575 - Created a PR to add myself to CODEOWNERS

File: .vscode/cspell.json
Comment (danieljurek) [inline .vscode/cspell.json:144]:
```suggestion
    "Rdma",
```

**Comment by rahuls-microsoft** [comment]
@ArthurMa1978 - Please suggest what needs to be updated.
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/computefleet/Azure.ResourceManager.ComputeFleet/assets.json#L5C11-L5C73 - What new GUID goes in there?

**Comment by rahuls-microsoft** [comment]
> @rahuls-microsoft :
> 
> Your account lacks the required public Azure GitHub organization required of an internal contributor. Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
> 
> * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_
> 
> You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.
> 
> Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

Made the Azure profile public and submitted Pr to Add to CodeOwners - https://github.com/Azure/azure-sdk-for-net/pull/46575
image

File: sdk/computefleet/Azure.ResourceManager.ComputeFleet/api/Azure.ResourceManager.ComputeFleet.netstandard2.0.cs
Comment (live1206) [inline sdk/computefleet/Azure.ResourceManager.ComputeFleet/api/Azure.ResourceManager.ComputeFleet.netstandard2.0.cs:3]: The API file is auto-generated, should not edit it.

File: .vscode/cspell.json
Comment (live1206) [inline .vscode/cspell.json:144]: probably add this only for your service, like L182-L186

**Comment by live1206** [comment]
Fix the CI failure, guide can be found in https://github.com/Azure/azure-sdk-for-net/pull/46541/checks?check_run_id=31491180031

File: sdk/computefleet/Azure.ResourceManager.ComputeFleet/CHANGELOG.md
Comment (live1206) [inline sdk/computefleet/Azure.ResourceManager.ComputeFleet/CHANGELOG.md:17]: remove this entry since you already added a new GA entry

Comment (rahuls-microsoft) [reply sdk/computefleet/Azure.ResourceManager.ComputeFleet/CHANGELOG.md:17]: Ok, will remove - ## 1.0.0-beta.2 (2023-11)

**Comment by live1206** [comment]
Since this is going GA, we should have a full test coverage for all operations, such as 
https://github.com/Azure/azure-sdk-for-net/blob/fb0b80d4c8fc99e9e964862f2a374d2bc28096f7/sdk/computefleet/Azure.ResourceManager.ComputeFleet/src/Generated/ComputeFleetResource.cs#L373C56-L373C87
Add more tests in scenario test beside https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/computefleet/Azure.ResourceManager.ComputeFleet/tests/Scenario/ComputeFleetCRUDTests.cs

File: sdk/computefleet/Azure.ResourceManager.ComputeFleet/api/Azure.ResourceManager.ComputeFleet.netstandard2.0.cs
Comment (rahuls-microsoft) [reply sdk/computefleet/Azure.ResourceManager.ComputeFleet/api/Azure.ResourceManager.ComputeFleet.netstandard2.0.cs:3]: ok, will remove it.

Comment (rahuls-microsoft) [reply sdk/computefleet/Azure.ResourceManager.ComputeFleet/api/Azure.ResourceManager.ComputeFleet.netstandard2.0.cs:3]: ok

File: .github/CODEOWNERS
Comment (jsquire) [inline .github/CODEOWNERS:1073]: Please revert this file.   It was properly updated in a separate PR.

Comment (rahuls-microsoft) [reply .github/CODEOWNERS:1073]: Restored to head - git restore -s HEAD .github\CODEOWNERS

Comment (jsquire) [reply .github/CODEOWNERS:1073]: Nope.  Still showing changes.   This file should not be included in the change set.

Comment (rahuls-microsoft) [reply .github/CODEOWNERS:1073]: Ok, what command should be used?

Comment (rahuls-microsoft) [reply .github/CODEOWNERS:1073]: I used -
git remote add mainSource https://github.com/rahuls-microsoft/azure-sdk-for-net.git
git fetch mainSource
git checkout main
git reset --hard mainSource/main  
git push origin main --force 

Is that fine?
How do I check it, as you checked? So, I can validate it before resolving it.

Comment (live1206) [reply .github/CODEOWNERS:1073]: I think the easiest way to revert is
- pull latest main to your local
- then run "git checkout origin/main .github/CODEOWNERS" under the repo root folder

Comment (rahuls-microsoft) [reply .github/CODEOWNERS:1073]: Also execute - git rebase mainSource/main - This should resolve the issue.
Else please let me know. Thanks

Comment (rahuls-microsoft) [reply .github/CODEOWNERS:1073]: > I think the easiest way to revert is
> 
> * fetch/pull latest origin/main to your local
> * switch to your working branch
> * then run "git checkout origin/main .github/CODEOWNERS" under the repo root folder

I have created a fork - https://github.com/rahuls-microsoft/azure-sdk-for-net/.
SO, taking too long time and showing lots of conflicts. Is there an easier way or should I just create branch from - https://github.com/Azure/azure-sdk-for-net. That may be an easier option.
Please suggest. Thanks for your help!

File: .vscode/cspell.json
Comment (live1206) [reply .vscode/cspell.json:144]: This is not updated though

File: .github/CODEOWNERS
Comment (live1206) [reply .github/CODEOWNERS]: I reverted it for you

File: .vscode/cspell.json
Comment (rahuls-microsoft) [reply .vscode/cspell.json:144]: Will move to service specific section, as suggested.- 
{
      "filename": "**/sdk/computefleet/**/*.cs",
      "words": [
        "Rdma"
      ]
    },

File: .github/CODEOWNERS
Comment (rahuls-microsoft) [reply .github/CODEOWNERS:1073]: Thanks

File: .vscode/cspell.json
Comment (rahuls-microsoft) [reply .vscode/cspell.json]: Updated as suggested.

File: .github/CODEOWNERS
Comment (rahuls-microsoft) [reply .github/CODEOWNERS:1073]: Sorry! How do I undo it?

Comment (live1206) [reply .github/CODEOWNERS]: I redid it.
Merge the latest from remote before you push to remote, instead of force push, it is error-prune with force push.

Comment (rahuls-microsoft) [reply .github/CODEOWNERS]: @live1206 - Please apply the changes. Thanks for yourr help.

**Comment by rahuls-microsoft** [comment]
> Since this is going GA, we should have a full test coverage for all operations, such as https://github.com/Azure/azure-sdk-for-net/blob/fb0b80d4c8fc99e9e964862f2a374d2bc28096f7/sdk/computefleet/Azure.ResourceManager.ComputeFleet/src/Generated/ComputeFleetResource.cs#L373C56-L373C87 Add more tests in scenario test beside https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/computefleet/Azure.ResourceManager.ComputeFleet/tests/Scenario/ComputeFleetCRUDTests.cs

I have added more tests.

The assets push -test-proxy push -a .\sdk\computefleet\Azure.ResourceManager.ComputeFleet\assets.json

**Giving Error: - Please suggest**
git remote set-url origin https://github.com/Azure/azure-sdk-assets
StdOut:
StdErr:
ExitCode: 0
Unhandled exception: Azure.Sdk.Tools.TestProxy.Common.Exceptions.HttpException: Invocation of "git add -A ." had a non-zero exit code -1.
StdOut:
StdErr: The following paths and/or pathspecs matched paths that exist
outside of your sparse-checkout definition, so will not be
updated in the index:
net/SessionRec.json
hint: If you intend to update such entries, try one of the following:
hint: * Use the --sparse option.
hint: * Disable or modify the sparsity rules.
hint: Disable this message with "git config advice.updateSparsePath false"

**Comment by rahuls-microsoft** [comment]
> > Since this is going GA, we should have a full test coverage for all operations, such as https://github.com/Azure/azure-sdk-for-net/blob/fb0b80d4c8fc99e9e964862f2a374d2bc28096f7/sdk/computefleet/Azure.ResourceManager.ComputeFleet/src/Generated/ComputeFleetResource.cs#L373C56-L373C87 Add more tests in scenario test beside https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/computefleet/Azure.ResourceManager.ComputeFleet/tests/Scenario/ComputeFleetCRUDTests.cs
> 
> I have added more tests.
> 
> The assets push -test-proxy push -a .\sdk\computefleet\Azure.ResourceManager.ComputeFleet\assets.json
> 
> **Giving Error: - Please suggest** git remote set-url origin https://github.com/Azure/azure-sdk-assets StdOut: StdErr: ExitCode: 0 Unhandled exception: Azure.Sdk.Tools.TestProxy.Common.Exceptions.HttpException: Invocation of "git add -A ." had a non-zero exit code -1. StdOut: StdErr: The following paths and/or pathspecs matched paths that exist outside of your sparse-checkout definition, so will not be updated in the index: net/SessionRec.json hint: If you intend to update such entries, try one of the following: hint: * Use the --sparse option. hint: * Disable or modify the sparsity rules. hint: Disable this message with "git config advice.updateSparsePath false"

@live1206 - Please suggest. Thanks for your help.

**Comment by rahuls-microsoft** [comment]
Can we release it next Tuesday - 10/22/2024?
The ARM Manifest rollout is in progress as a hotfix, so we should complete it in the next 2 days.

Thanks.

**Comment by SahilArora92** [comment]
Please fix the build errors.

**Review by live1206 — APPROVED** [review]

**Pull request merged**
